chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -936,6 +936,29 @@ impl PeerManagerActor {
             .collect::<Vec<_>>()
     }
 
+    // Get peers that are potentially unreliable and we should avoid routing messages through them.
+    // Currently we're picking the peers that are NOT the highest peers.
```

Comment:
Update comments, I think the code actually implements "peers that are too much behind the highest peers".

Reply:
done.